Formula/ocrmypdf.rb

```diff
@@ -27,7 +27,7 @@ class Ocrmypdf < Formula
   depends_on "libpng"
   depends_on "pngquant"
   depends_on "pybind11"
-  depends_on "python@3.8"
+  depends_on "python@3.9"
```

Comment:
```
  * ocrmypdf contains conflicting version recursive dependencies:
      python@3.8, python@3.9
```

We should not migrate individual formulas to Python 3.9, unless they have no dependency and no dependent involving Python.

Reply:
Sounds about right, let me revert this change.